src/udev/hidraw_id/hidraw_id_desc.h

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#pragma once
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+typedef enum HidrawIdDescResult {
+        HIDRAW_ID_SECURITY_TOKEN = 1 << 0,
+        HIDRAW_ID_U2F_TOKEN      = 1 << 1,
```

Comment:
As @poettering commented, please make this simple enum, that is please drop `= 1 << x`.

Reply:
or drop it altogether, the current code will only ever set HIDRAW_ID_U2F_TOKEN anyway, no? why have the other bit?